package.json

```diff
@@ -80,10 +80,6 @@
             "plugin:@taiga-ui/experience/entry-points"
         ],
         "rules": {
-            "i18n/no-greek-character": "off",
```

Comment:
Russian I understand, so there's no Russian "c" hidden in code by accident, but what's wrong with others?

Reply:
`but what's wrong with others`

prevent contribution like this

<img width="773" alt="image" src="https://github.com/user-attachments/assets/d8a501ed-957f-43de-909a-b7429d597667">
<img width="945" alt="image" src="https://github.com/user-attachments/assets/27775dae-8dfb-4c78-8726-c8de40abd2fa">

Reply:
fixed